### PR TITLE
Dummy PR to try to trigger events.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

**Comment by thomasgtaylor** [comment]
comment

**Review by thomasgtaylor — APPROVED** [review]
LGTM

**Review by dhermes — COMMENTED** [review]
Oh a comment is different than approve

File: README.md [hunk 1]
```diff
@@ -34,3 +34,5 @@ For links (which may have expired) to the original runs, see the
 [11]: _examples/schedule/1464380337
 [12]: https://github.com/dhermes/actions-playground/actions/runs/1464429734
 [13]: _examples/workflow_run/1464429734
+
+<!-- This will not even show up -->
```
Comment (dhermes) [inline README.md]: What about an inline comment?

**Pull request merged**